featurebyte/session/bigquery.py

```diff
@@ -284,6 +287,25 @@ def get_additional_execute_query_kwargs(self) -> dict[str, Any]:
         )
         return {"job_config": job_config}
 
+    @property
+    def dataset_ref(self) -> DatasetReference:
+        return DatasetReference(project=self.project_name, dataset_id=self.dataset_name)
+
+    @staticmethod
+    def _execute_query(cursor: Any, query: str, **kwargs: Any) -> Any:
+        job_id = str(uuid.uuid4())
```

Comment:
specify job id for request so we don't need to wait for job handle to be available after query

Reply:
Is there any concern over this note on setting the job id manually? https://github.com/googleapis/python-bigquery/blob/e29b987956e5d80541ad9a573e902938a1373bda/google/cloud/bigquery/dbapi/cursor.py#L180-L183

Reply:
When not provided the bigquery client code will create one the same way as here:
https://github.com/googleapis/python-bigquery/blob/e29b987956e5d80541ad9a573e902938a1373bda/google/cloud/bigquery/_job_helpers.py#L84